airflow/utils/db.py

```diff
@@ -284,6 +284,10 @@ def initdb():
         Connection(
             conn_id='azure_container_instances_default', conn_type='azure_container_instances',
             extra='{"tenantId": "<TENANT>", "subscriptionId": "<SUBSCRIPTION ID>" }'))
+    merge_conn(
```

Comment:
Are you sure you need a new connection? Google Cloud Platform (`google_cloud_platform`) does not create a separate connection for each of your services. If you create a separate connection, each user will have to enter the credentials to Azure again.

Reply:
@mik-laj what would be the alternative of this?

Reply:
I think you should create one connection for all Azure services ex. `azure_default`. Look at: https://github.com/apache/airflow/pull/4494/files#r249418220